models/iaf_chxk_2008_implicit.nestml

```diff
@@ -73,13 +73,13 @@ neuron iaf_chxk_2008_implicit:
   internals:
     # Impulse to add to DG_EXC on spike arrival to evoke unit-amplitude
     # conductance excursion.
-    PSConInit_E real = 1.0 * e / tau_syn_ex
+    PSConInit_E real = 1.0 ms * e / tau_syn_ex
```

Comment:
I'm very sorry this comes so late (I must say I completely forgot this PR) but I just noticed the combination of lines 76/89 and [120](https://github.com/PTraeder/nestml/blob/c0a589c34b2c7109689d6fc44c96ddd0c2fd1943/models/iaf_chxk_2008_implicit.nestml#L120)/121 is redundant (therefore less clear) in terms of units: explicit units are used twice because ``PSConInit_I/E`` is ``real``, whereas it could directly be ``nS``. 
This would lead to:

    PSConInit_E nS = 1.0 nS * e / tau_syn_ex
    PSConInit_I nS = 1.0 nS * e / tau_syn_ex

then

    g_ex' += spikesExc * PSConInit_E
    g_in' += spikesInh * PSConInit_I